luigi/task.py

```diff
@@ -146,6 +146,8 @@ class MyTask(luigi.Task):
     #: Only works when using multiple workers.
     worker_timeout = None
 
+    config = {}
```

Comment:
You are right, more spesific and readable naming should be used. But this contains some configuration from luigi config. It may seems only for `retry-policy` at the moment. But I thought that this feature may includes another configuration besides `retry-policy` by the end of the day. I don't know, `email-sender`, maybe `email-prefix` or another suitable config. So maybe naming should not be related with retry-policy. What would you think about it, am I missing it?